### PR TITLE
doc: Documentation for AVB Streamhandler Systemd watchdog

### PR DESCRIPTION
**Pull request by tvyas**
Add the documentation to public/docs.

Signed-off-by: Tarun Vyas <tarun.vyas@intel.com>

### CODE REVIEW

**Review by edersondisouza — COMMENTED** [review]
Looks OK, but maybe this should be a section inside `01_icd.md` file? It appears that all high level information about avb-sh goes there, so let's keep it. What do you think?

**Comment by tvyas** [comment]
Errr, yea, I should look around a bit more next time, before creating a separate doc.

**Comment by tvyas** [comment]
Done

**Comment by tvyas** [comment]
Looks like I added it in the wrong place (all the way at the end of the doc)

File: public/doc/01_icd.md [hunk 1]
```diff
@@ -1303,7 +1303,45 @@ FAQ
   A: No, there are no packets. Since the packet transmission rate is adaptive to the payload bit rate, if the payload is 0, the packet rate is 0 as well.
   You will have to feed data to the local streaming interface to generate packets on the network. There is a debug application that can be used to generate dummy video data.
 
-
+### Systemd Watchdog
```
Comment (edersondisouza) [inline public/doc/01_icd.md]: Maybe use the "@section" thing header of other sections of this file? I guess that there's some doxygen trick in place (but we can't test, as documentation build is disabled =/ ).

Comment (tvyas) [reply public/doc/01_icd.md]: So I guess we want to leave this all the way at the end of the doc ? Adding it elsewhere feels like breaking the continuity of the doc.

Comment (edersondisouza) [reply public/doc/01_icd.md]: Maybe before the FAQ? FAQ being the last one makes sense.

Comment (tvyas) [reply public/doc/01_icd.md]: I'll skip the @section part, b/c then this will become a section of the last _chapter_, which is _Registries_ in this case. So, I'll just make it a new chapter before FAQ.

Comment (edersondisouza) [reply public/doc/01_icd.md]: Chapter? Really? Where is this defined? Guess I did something wrong on my PR adding stuff here then... hehe. Anyway, yep, it should be on the same level as FAQ.

Comment (tvyas) [reply public/doc/01_icd.md]: Sorry I meant the L1 header :D, couldn't find the right word at the moment.

Comment (tvyas) [reply public/doc/01_icd.md]: Done !

File: public/doc/01_icd.md [hunk 2]
```diff
@@ -1303,8 +1342,6 @@ FAQ
   A: No, there are no packets. Since the packet transmission rate is adaptive to the payload bit rate, if the payload is 0, the packet rate is 0 as well.
   You will have to feed data to the local streaming interface to generate packets on the network. There is a debug application that can be used to generate dummy video data.
 
-
-
```
Comment (edersondisouza) [inline public/doc/01_icd.md]: This is unrelated.

File: public/doc/01_icd.md [hunk 3]
```diff
@@ -1275,6 +1275,45 @@ When starting the Stream Handler via *systemd* these variables are set by the se
 ![Figure: Sequence to create a transmit stream](./images/MSC_Avb_Control_Establish_Streams.png "Creating a network stream associated with a local source")
 ![Figure: Sequence to remove a transmit stream](./images/MSC_Avb_Control_Remove_Stream.png "Removing a network stream")
 
+# Systemd Watchdog
```
Comment (edersondisouza) [inline public/doc/01_icd.md]: Sure that you don't need a
```
#################################################################################
@section watchdog Systemd Watchdog
#################################################################################
```

here? I guess that this section would appear an alien in the final document...

Comment (tvyas) [reply public/doc/01_icd.md]: Using a 
```
#################################################################################
@section watchdog Systemd Watchdog
#################################################################################
```
would mean that it falls under the _Registries_ Header, wouldn't it ?

Comment (edersondisouza) [reply public/doc/01_icd.md]: Nope.

File: public/doc/01_icd.md [hunk 4]
```diff
@@ -1275,6 +1275,48 @@ When starting the Stream Handler via *systemd* these variables are set by the se
 ![Figure: Sequence to create a transmit stream](./images/MSC_Avb_Control_Establish_Streams.png "Creating a network stream associated with a local source")
 ![Figure: Sequence to remove a transmit stream](./images/MSC_Avb_Control_Remove_Stream.png "Removing a network stream")
 
+#################################################################################
+@section watchdog Systemd Watchdog
+#################################################################################
+Systemd Watchdog
```
Comment (edersondisouza) [inline public/doc/01_icd.md]: As there's a title over there, I guess you don't need this here (this extra "Systemd Watchdog").

Comment (tvyas) [reply public/doc/01_icd.md]: Ooops !

**Review by edersondisouza — APPROVED** [review]
